tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathDescriptions.md

```diff
@@ -0,0 +1,1115 @@
+
+###
+
+- `^//ldml/localeDisplayNames/annotationPatterns/annotationPattern[@type="([^"]*)"]`
+
+The pattern to be used when displaying a {1}. For more information, please see [Locale Patterns].
+
+###
+
+- `^//ldml/characters/stopwords/stopwordList\[@type="([^"]*)"]`
+
+The words that should be ignored in sorting in your language. For more information, see [Units Misc Help].
+
+###
+
+- `^//ldml/dates/timeZoneNames/zone\[@type="([^"]*)"]/([^/]*)/(.*)`
+
+Override for the $3-$2 timezone name for $1. For more information, see [Time Zone City Names].
+
+###
+
+- `^//ldml/typographicNames/axisName[@type="([^"]*)"]`
+
+A label for a typographic design axis, such as “Width” or “Weight”. For more information, see [Typography].
+
+###
+
+- `^//ldml/typographicNames/styleName[@type="([^"]*)"][@subtype="([^"]*)"]`
+
+A label for a typographic style, such as “Narrow” or “Semibold”. For more information, see [Typography].
+
+###
+
+- `^//ldml/typographicNames/featureName[@type="([^"]*)"]`
+
+A label for a typographic feature, such as “Small Capitals”. For more information, see [Typography].
+
+###
+
+- `^//ldml/characterLabels/characterLabelPattern\[@type="([^"]*)"]\[@count="([^"]*)"]`
+
+A label for a set of characters that has a numeric placeholder, such as “1 Stroke”, “2 Strokes”. For more information, see [Character Labels].
+
+###
+
+- `^//ldml/characterLabels/characterLabelPattern\[@type="([^"]*)"]`
+
+A modifier composed with a label for a set of characters. For more information, see [Character Labels].
+
+###
+
+- `^//ldml/characterLabels/characterLabel\[@type="([^"]*)"]`
+
+A label for a set of characters. For more information, see [Character Labels].
+
+###
+
+- `^//ldml/annotations/annotation\[@cp="([^"]*)"]\[@type="tts"]`
+
+A name for a character or sequence. For more information, see [Short Character Names].
+
+###
+
+- `^//ldml/annotations/annotation\[@cp="([^"]*)"]`
+
+A set of keywords for a character or sequence. For more information, see [Short Character Names].
+
+## References
+
+<!--
+This section is appended to every markdown fragment.
+-->
+
+[Characters]: https://cldr.unicode.org/translation/characters
+[Character Labels]: https://cldr.unicode.org/translation/characters/character-labels
+[Compound Units]: https://cldr.unicode.org/translation/units/unit-names-and-patterns#compound-units
+[Country Names]: https://cldr.unicode.org/translation/displaynames/countryregion-territory-names
+[Currency Names]: https://cldr.unicode.org/translation/currency-names-and-symbols
+[Date Time]: https://cldr.unicode.org/translation/date-time/date-time-names#h.ewzjebmpoi4k
+[Date Time Names]: https://cldr.unicode.org/translation/date-time/date-time-names
+[DATE_TIME_NAMES_CYCLIC]: https://cldr.unicode.org/translation/date-time/date-time-names#h.h0vy2eyzcj0n
+[Date Time Fields]: https://cldr.unicode.org/translation/date-time/date-time-names#h.8wfk3599ck9x
```

Comment:
When you looked up the #h. fragments to get the header names, you should have also fixed the fragment to be right. Right now it fails and just goes to the top of the page.
```suggestion
[Date Time Fields]: https://cldr.unicode.org/translation/date-time/date-time-names#date-field-names
```

Reply:
didn't look them up, just updated the titles.